lib/rules/use-tokens/__tests__/index.test.js

```diff
@@ -34,6 +34,17 @@ describe('Spacing token tests', () => {
     expect(warnings).toHaveLength(0);
   });
 
+  it('passes when using multiples of spacing', async () => {
+    const {
+      results: [{ warnings }],
+    } = await lint({
+      code: 'a { width: $bpk-spacing-md * 2; font-size: 14rem }',
```

Comment:
This might be obvious, but why doesn't this fail on `font-size: 14rem`? Surely that should use a token too?

Reply:
Because currently this linter doesn't process font sizes - that is something to build on in the future :)

Reply:
As this project mirrors that of the `eslint-plugin-backpack` that also does not lint on font tokens the original mvp was just to mirror but future plans will add it in at a later point in time